ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/parser/TestDirective.java

```diff
@@ -79,21 +83,13 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "TestDirective{"
-        + "type=" + type
-        + ", contents='" + contents + '\''
-        + '}';
-  }
-
-  public NodeLocation getLocation() {
-    return location;
+    return "--@" + type.getTypeName() + ": " + contents;
   }
 
   public enum Type {
     TEST("test"),
     EXPECTED_ERROR("expected.error"),
     EXPECTED_MESSAGE("expected.message"),
-    ENABLED("enabled"),
```

Comment:
removed this because it's not super easy to implement cleanly given the iterative nature of the tests. since the sql file supports multiline comments, it's easy enough to disable a test like this:
```sql
/* (disabled)
--@test: foo bar
CREATE STREAM foo...;
INSERT INTO ...;
ASSERT VALUES ...;
*/
```